infrastructure/modules/resource_templates/aks/main.tf

```diff
@@ -0,0 +1,20 @@
+data "azurerm_resource_group" "resource_group" { name = var.resource_group_name }
+
+resource "azurerm_kubernetes_cluster" "aks" {
+  name                = var.aks_name
+  location            = data.azurerm_resource_group.resource_group.location
+  resource_group_name = data.azurerm_resource_group.resource_group.name
+  dns_prefix          = var.aks_name
+
+  default_node_pool {
+    name       = "default"
```

Comment:
Perhaps not for now - but this feels like environment specific config or a default variable perhaps.

Reply:
Indeed you are correct, this infra is just baseline for now but one to keep in mind when we start to abstract the different environments

Reply:
@guypritchard you'll see some more configuration in an upcoming PR very soon